External/Plugins/HaXeContext/Context.cs

```diff
@@ -577,6 +580,21 @@ public override void RemoveClassCompilerCache()
         }
         #endregion
 
+        #region SDK
+        private InstalledSDK GetCurrentSDK()
+        {
+            return hxsettings.InstalledSDKs.FirstOrDefault(sdk => sdk.Path == currentSDK);
```

Comment:
`FirstOrDefault` sounds super useful but it sounds like it will never return null - or can it?